### PR TITLE
Custom delimiters for B__E--M

### PR DESCRIPTION
**Pull request by agudulin**
re #18

### CODE REVIEW

File: src/plugins/makeBlock.js [hunk 1]
```diff
@@ -20,4 +20,4 @@
  *   // => 'block__element'
  *
  */
-export const maker = (block, _) => block
+export const maker = (block, _props, _options) => block
```
Comment (robinpokorny) [inline src/plugins/makeBlock.js]: Why?

File: src/index.js [hunk 2]
```diff
@@ -8,18 +8,22 @@ import * as pluginStates from './plugins/makeStates'
 import filterObject from './filter-object'
 
 const basicPlugins = [pluginBlock, pluginModifiers, pluginOriginalClass, pluginStates]
+const defaultDelimeter = {
```
Comment (robinpokorny) [inline src/index.js]: Shouldn't it be plural?

File: src/plugins/makeBlock.js (hunk 1 above)
Comment (agudulin) [reply src/plugins/makeBlock.js]: I want to show that the function takes 3 arguments here.

File: src/index.js (hunk 2 above)
Comment (agudulin) [reply src/index.js]: yep :)

**Review by robinpokorny — APPROVED** [review]

**Pull request merged**